### PR TITLE
added typings for jQuery Mask plugin

### PR DESCRIPTION
**Pull request by avidenic**
Please fill in this template.

- [x] Make your PR against the `master` branch.
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped#common-mistakes).
- [x] Run `tsc` without errors.
- [x] Run `npm run lint package-name` if a `tslint.json` is present.

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not provide its own types, and you can not add them.
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `npm run new-package package-name`, not by basing it on an existing project.

### CODE REVIEW

**Comment by andy-ms** [comment]
Needs a `tslint.json` containing `{ "extends": "../tslint.json" }`. (And I can tell there will be lint errors, so remember to run the linter!)

**Comment by avidenic** [comment]
I messed up fork sync and accidentally closed PR when mending the mess up. It should be ok now.

**Comment by andy-ms** [comment]
The pull requests still has more commits than just yours. Could you rebase to get rid of them? (Remember to make a backup of your changes first.)

**Comment by avidenic** [comment]
Now it should be really ok. I guess I should not sync until pull request is complete.

File: jquery-mask-plugin/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "target": "es6",
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictNullChecks": false,
```
Comment (andy-ms) [inline jquery-mask-plugin/tsconfig.json]: Make this `true`.

**Pull request merged**